samples/meetings-share-to-stage-signing/csharp/README.md

```diff
@@ -13,13 +13,30 @@ createdDate: "09-03-2022 15:56:00"
 ---
 
 # In-Meeting Document Signing
-This proof of concept demonstrates the use case of sharing a document on Teams Share to Stage with live signing feature in a meeting.
+Meeting Signing is a Teams sample app that demonstrates the scenario of enabling users to sign documents in real time during meetings making possible to review and sign documents in a single session. This sample will let users electronically sign documents by certifying their identity. Many kind of documents like NDAs, Purchase Orders and Contracts can be signed during meetings where real time collaboration happens. This template can be used for multiple scenarios such as approve PRs, Purchase Orders, Document Changes 
```

Comment:
What are you trying to get across with "Many kind of documents like NDAs, Purchase Orders and Contracts can be signed during meetings where real time collaboration happens. This template can be used for multiple scenarios such as approve PRs, Purchase Orders, Document Changes"?

Would something like the following cover what you are thinking? I want to be sure we call out what is possible today and what would require work on their part.
"This sample allows for example documents, including Purchase Agreements and Purchase Orders, to be signed when real time collaboration happens in a Teams meeting. This template is also applicable, and could be adapted for other scenarios like reviewing and approving code reviews or collaborative reviewing of documents"

Do we have an alternative use case than "Collaborative reviewing of documents"? I wasn't able to think of one off the top of my head.